collaborative_practice.js

```diff
@@ -19,6 +20,15 @@ const listUsers = (users) => {
     console.log(output);
 };
 
+const searchUsers = (firstName, lastName, users) => {
+    const output = users
+        .map(user => (user.firstName === firstName && user.lastName === lastName) 
```

Comment:
We need to search based on a single input and check if that input is a substring of firstName or lastName